app/serializers/request-config.js

```diff
@@ -0,0 +1,9 @@
+import V3Serializer from 'travis/serializers/v3';
+
+export default V3Serializer.extend({
+  normalize(modelClass, resourceHash) {
+    resourceHash.id = new Date().getTime();
```

Comment:
Does `resourceHash.id` ever already exist coming into this function? If so, it may make sense to do something like `resourceHash.id = resourceHash.id || new Date().getTime();`. If not, feel free to ignore this

Reply:
no, it won't ever have an id, @sorta. this endpoint expands the posted request config _as if_ it would actually be processed by Gatekeeper/Yml, and only then _would_ be persisted.  so there's pretty much no way the client would ever get an id on this. i believe the key is present in the api payload because our v3 api abstraction makes it extremely hard to omit it, when using all the default framework tooling :)

Reply:
Cool, makes sense :)